cmd/root.go

```diff
@@ -80,16 +75,42 @@ var RootCmd = &cobra.Command{
 		fmt.Println()
 		err := tuiUtils.PrintStyledText("ATMOS")
 		if err != nil {
-			u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+			u.LogErrorAndExit(err)
 		}
 
 		err = e.ExecuteAtmosCmd()
 		if err != nil {
-			u.LogErrorAndExit(schema.AtmosConfiguration{}, err)
+			u.LogErrorAndExit(err)
 		}
 	},
 }
 
+func setupLogger(atmosConfig *schema.AtmosConfiguration) {
+	switch atmosConfig.Logs.Level {
+	case "Trace":
+		log.SetLevel(log.DebugLevel)
+	case "Debug":
+		log.SetLevel(log.DebugLevel)
+	case "Info":
+		log.SetLevel(log.InfoLevel)
+	case "Warning":
+		log.SetLevel(log.WarnLevel)
+	case "Off":
+		log.SetOutput(io.Discard)
+	default:
+		log.SetLevel(log.InfoLevel)
+	}
+
+	if atmosConfig.Logs.File != "/dev/stderr" {
```

Comment:
If treating `/dev/stderr` differenly, maybe it should be all `/dev` devices? E.g. `/dev/null`, and `/dev/stdout`

Reply:
Well, for some unknown reason `atmos` is defaulting the log "file" to `/dev/stderr`, so basically I just am saying "if it's anything but the default, then set the log file to what is provided"